### PR TITLE
Ft automatic installation parsec

### PR DESCRIPTION
**Pull request by mahmoudElkarargyBS**
### Authored By

@mahmoud.elkarargy 

### Target Branches

- **Branch Out From**: Climate-Emulator
- **Merge Into**: Climate-Emulator

### Implementation Details

Installing of hicma-x, starsh, hcore

### Checklist

- [x] The feature has been fully implemented.
- [x] Minor fixing of documentation requested by sameh.
- [x] Documentation has been updated (if applicable).
- [x] All pipelines are passing.
- [x] Installation is tested on Vulture and Hen

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -18,24 +18,36 @@ cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 cmake_policy(SET CMP0048 NEW)
 
 # Set project options
-option(USE_CUDA "Use Cuda, if available" false)
```
Comment (SAbdulah) [inline CMakeLists.txt:21]: Let's use CUDA instead of Cuda

File: cmake/FindHICMA-X.cmake [hunk 2]
```diff
@@ -0,0 +1,108 @@
+
+# Copyright (c) 2017-2024 King Abdullah University of Science and Technology,
+# All rights reserved.
```
Comment (SAbdulah) [inline cmake/FindHICMA-X.cmake:3]: Did you write this? Or getting it from another place?

Comment (SAbdulah) [reply cmake/FindHICMA-X.cmake:3]: I mean the while file

**Review by SAbdulah — COMMENTED** [review]
Some minors to fix, then I can approve the pull request @mahmoudElkarargyBS

File: cmake/FindHICMA-X.cmake (hunk 2 above)
Comment (mahmoudElkarargyBS) [reply cmake/FindHICMA-X.cmake:3]: Yes, I wrote it

**Pull request merged**